core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ConfigResource.java

```diff
@@ -106,7 +106,7 @@ public ConfigUpdateRequest updateSubjectLevelConfig(
       @NotNull ConfigUpdateRequest request) {
     CompatibilityLevel compatibilityLevel =
         CompatibilityLevel.forName(request.getCompatibilityLevel());
-    if (compatibilityLevel == null) {
+    if (request.getCompatibilityLevel() != null && compatibilityLevel == null) {
       throw new RestInvalidCompatibilityException();
```

Comment:
The ConfigUpdateRequest has several parameters, all of them optional.  The previous check would fail if the compatibilityLevel was null or unrecognized.  The new check only fails if the compatibilityLevel is unrecognized, allowing it to be optional.

Reply:
Got it. Should we also update the API doc (eg. summary and description) for the endpoints since it's been extended to update multiple configs?